vcx/wrappers/python3/demo/faber.py

```diff
@@ -24,14 +27,15 @@
 # 'wallet_key': encryption key for encoding wallet
 # 'payment_method': method that will be used for payments
 provisionConfig = {
-    'agency_url': 'http://localhost:8080',
-    'agency_did': 'VsKV7grR1BUE29mG2Fm2kX',
-    'agency_verkey': 'Hezce2UWMZ3wUhVkh2LfKSs8nDzWwzs2Win7EzNN3YaR',
-    'wallet_name': 'faber_wallet',
+    'agency_url': 'https://indy-dev.datawallet.origincert.com/mediator',
+    'agency_did': 'X2b2m5JyzeroKKB4X44VHY',
+    'agency_verkey': 'HNDzQx2JSytjWMcfqWoTyCtMCyfL7dvKcQfDGvjaVhNY',
+    'wallet_name': 'faber_wallet9',
     'wallet_key': '123',
     'payment_method': 'null',
-    'enterprise_seed': '000000000000000000000000Trustee1',
-    'protocol_type': '3.0',
+    'enterprise_seed': 'bHjydDGBjNBTp5cjGCK8bzbzvW4rR9kb',
+    'protocol_type': '2.0',
+    'communication_protocol': 'aries',
```

Comment:
For me, the TAA works with the original agency provision and dummy cloud agency, which `README.md` directs the user to use. Why this change?

Reply:
I agree with Mirgee the demo configuration should not have by default have external dependencies (external ledger, 3rd party cloud agent, someone's seed, ...)